doc/source/admin/galaxy_options.rst

```diff
@@ -4491,4 +4491,19 @@
 :Type: str
 
 
+~~~~~~~~~~~~~~~~~~~~~~~~~
+``data_table_connection``
```

Comment:
Can we do ``data_table_database_connection`` for consistency?

Reply:
I've renamed it, thanks for the suggestion!

Reply:
I've renamed it, thanks for the suggestion!